tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/js/output-console/service-console-list-manager.js

```diff
@@ -217,21 +224,45 @@ define(['log', 'jquery', 'lodash', 'output_console_list', 'workspace', 'service_
                             opts.application.outputController.hideAllConsoles();
                         }
                         var logMessage = "Connection closed, backend is unavailable! " +
-                            "Refresh to reconnect the console.";
+                            "Please try reconnecting the console "+
+                            "by pressing the reload button "+
+                            "when backend is available.";
                         var message = {
                             "type": "ERROR",
                             "message": logMessage
                         };
+                        enableLink($(_.get(opts, 'reloadConsoleBtn'))[0]);
                         console.println(message);
-                        // not retrying
-                        // console.log('Editor console service has terminated. Reconnect will be attempted in 1 second.',
-                        //     error.reason);
-                        // setTimeout(function () {
-                        //     opts.application.outputController.initiateLogReader(opts);
-                        // }, 1000);
                     };
                 }
-
             });
         return OutputConsoleList;
     });
+
+/**
+ *enable the reconnect button when  backend is unavailable.
+ * @param link
+ */
+let enableLink = function (link) {
```

Comment:
dont use let

Reply:
13712b2ee95c1b9ae9345ac8be85cefbbe170fd9 this commit fix above requirement